### ISSUE TITLE
GL.GL_MULTISAMPLE not supported on Windows and MacOS

### ISSUE THREAD

**Issue by ziyaointl** [p5py/p5#399]
**Describe the bug**
`GL.glEnable(GL.GL_MULTISAMPLE)` causes p5py to crash under the skia renderer in MacOS and Windows.

**To Reproduce**
Run the any p5 script with skia as renderer.

**Expected behavior**
p5py should not crash

**Screenshots**
N/A

**System information:**
MacOS
 - p5 release (version number or latest commit): 6f0d09632ce67977741f1c2bf3fa68f6d99bb463
 - Python version: Python 3.9.13
 - Operating system: MacOS Montery 12.4

Windows
 - p5 release (version number or latest commit): 6f0d09632ce67977741f1c2bf3fa68f6d99bb463
 - Python version: Python 3.10.7
 - Operating system: Windows 10 Pro 20H2

**Additional context**
Windows stack trace:
```bash
Traceback (most recent call last):
  File "test.py", line 22, in <module>
    run(renderer='skia')
  File "p5\sketch\userspace.py", line 231, in run
    p5.sketch.start()
  File "p5\sketch\Skia2DRenderer\base.py", line 159, in start
    self.window = self.glfw_window()
  File "p5\sketch\Skia2DRenderer\base.py", line 80, in glfw_window
    GL.glEnable(GL.GL_MULTISAMPLE)
  File "OpenGL\platform\baseplatform.py", line 415, in __call__
    return self( *args, **named )
  File "OpenGL\error.py", line 230, in glCheckError
    raise self._errorClass(
OpenGL.error.GLError: GLError(
        err = 1282,
        description = b'invalid operation',
        baseOperation = glEnable,
        cArguments = (GL_MULTISAMPLE,)
)
```
MacOS stack trace:
```bash
$ python -X faulthandler issue352.py
Fatal Python error: Segmentation fault

Current thread 0x0000000111041600 (most recent call first):
  File "OpenGL/platform/baseplatform.py", line 415 in __call__
  File "p5/sketch/Skia2DRenderer/base.py", line 80 in glfw_window
  File "p5/sketch/Skia2DRenderer/base.py", line 159 in start
  File "p5/sketch/userspace.py", line 231 in run
  File "issue352.py", line 40 in <module>
[1]    65875 segmentation fault  python -X faulthandler issue352.py
```

We may need to find another way to enable antialiasing in skia.

**Comment by tushar5526** [comment]
I think I should try setting the settings after creating a context, and it might work fine. 
https://stackoverflow.com/questions/68527174/invalid-operation-when-glenablegl-multisample-in-pyopengl

**Comment by ziyaointl** [comment]
Thanks! I confirm that antialiasing works on windows if we removed the 3 lines:
![Capture](https://user-images.githubusercontent.com/18119047/192658232-78157543-d438-4308-af21-bb3733681bb2.PNG)

**Comment by ziyaointl** [comment]
On MacOS, removing the three lines also worked - though there seems to be a separate issue of the image not being the correct size.
<img width="919" alt="Screen Shot 2022-09-27 at 8 05 20 PM" src="https://user-images.githubusercontent.com/18119047/192658508-724fba17-40c8-4cbb-b870-56c4964eb665.png">

**Comment by tushar5526** [comment]
The macos issue is due to the retina display. I will recheck the code to see if I am initialising values properly. Somewhere framebuffers are not being configured properly

**Comment by ziyaointl** [comment]
Closing issue since this was resolved in #400
